p2p/simulations/network.go

```diff
@@ -440,6 +469,17 @@ func (self *Network) DidDisconnect(one, other *adapters.NodeId) error {
 	return nil
 }
 
+// Send(senderid, receiverid) sends a message from one node to another
+func (self *Network) Send(senderid, receiverid *adapters.NodeId, msgcode uint64, protomsg interface{}) {
+	msg := &Msg{
```

Comment:
but we would want the sender and receiver here, right? Need to output it to the visualization layer. And in "conn" events the "conn" object structure is already defined through to client. That's why I mentioned that.